sdk/keyvault/Azure.Security.KeyVault.Keys/README.md

```diff
@@ -361,7 +361,7 @@ When you submit a pull request, a CLA-bot will automatically determine whether y
 This project has adopted the [Microsoft Open Source Code of Conduct][code_of_conduct]. For more information see the Code of Conduct FAQ or contact opencode@microsoft.com with any additional questions or comments.
 
 <!-- LINKS -->
-[API_reference]: https://azure.github.io/azure-sdk-for-net/api/Azure.Security.KeyVault.Keys.html
+[API_reference]: https://azuresdkdocs.blob.core.windows.net/$web/dotnet/Azure.Security.KeyVault.Keys/4.0.0-preview.6/api/index.html
```

Comment:
This will not be preview.6.

Reply:
Very good point. For now I've asked @chidozieononiwu to limit this PR to Identity as we know the version of it and plan to release it right now.